integration/spark-interfaces-scala/src/main/scala/io/openlineage/spark/builtin/scala/v1/relation.scala

```diff
@@ -0,0 +1,34 @@
+package io.openlineage.spark.builtin.scala.v1
+
+import io.openlineage.client.utils.DatasetIdentifier
+
+/** Trait to be implemented for extension's classes extending
+  * `org.apache.spark.sql.sources.BaseRelation`. Implementing it allows
+  * extracting lineage from such objects. Implementing `getNamespace` and
+  * `getName` within the `Dataset` is obligatory.
+  */
```

Comment:
In the comparison to previous docs, I really like those 👍 

Just replace 

```
within the `Dataset`
```

to 

```
within the `DatasetIdentifier`
```